src/index.ts

```diff
@@ -17,13 +17,15 @@ export type Options = {
   strength?: number
   path?: string
   mnemonic?: string | null
+  passphrase?: string | null
 }
 
 const defaultOptions = {
   // default path is BIP-44, where depth 5 is the address index
   path: "m/44'/60'/0'/0",
   strength: 256,
   mnemonic: null,
+  passphrase: "",
```

Comment:
Feels like defaulting this to null is more in keeping with overall intent, even though it doesn't make a difference rn.

Reply:
Understandable. I made it this way so that even if the underlying library changed we would still be using the default defined by the BIP